src/features/commands/report.js

```diff
@@ -60,10 +61,15 @@ module.exports = {
                 ephemeral: true
             });
         }
-
         if (!interaction.guild.rulesChannelId) {
             return await interaction.reply({
-                content: `Reports are supported only in community guilds.`,
+                content: 'Reports are supported only in community guilds.',
+                ephemeral: true
+            });
+        }
+        if (reportedMessage.author.bot) {
+            return await interaction.reply({
+                content: 'Message reports can only be used on members.',
```

Comment:
Bots technically count as members, so maybe modify the message to "Message reports cannot be used on bots."

Reply:
That's what it was supposed to be, my brain must've paused while writing that LOL